packages/driver/cypress/integration/e2e/uncaught_errors_spec.js

```diff
@@ -158,6 +158,19 @@ describe('uncaught errors', () => {
     cy.visit('/fixtures/global-error.html')
   })
 
+  it('global unhandledrejection', (done) => {
+    cy.once('uncaught:exception', (err) => {
```

Comment:
Let's have unhandled rejections emit their own event, `unhandled:rejection`, instead of overloading `unhandled:exception`. I think the change should be clearer that way and going forward.